### PR TITLE
Added Gradient so doesn't look boring

### PR DESCRIPTION
**Pull request by Lachsfilet**
# Description

Added a gradient to the Courses header

## Type of change

Style Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] The code works fine on my machine.
- [ ] I have modified the build steps if changed.

### CODE REVIEW

File: app/(root)/(routes)/roadmap/page.tsx [hunk 1]
```diff
@@ -219,9 +219,9 @@ export default function Roadmap() {
             <div>
                 <h1 className="text-6xl text-center font-bold mx-4"> 
                     {!session ? (
-                        <p>Roadmap</p>
+                        <p className="bg-gradient-to-r from-blue-600 via-green-500 to-indigo-400 inline-block text-transparent bg-clip-text">Roadmap</p>
                     ): (
-                        <p>{session?.user.name}'s Roadmap</p>
+                        <p className="bg-gradient-to-r from-blue-600 via-green-500 to-indigo-400 inline-block text-transparent bg-clip-text">{session?.username}'s Roadmap</p>
```
Comment (dpaulos6) [inline app/(root)/(routes)/roadmap/page.tsx]:
```suggestion
                        <p className="bg-gradient-to-r from-blue-600 via-green-500 to-indigo-400 inline-block text-transparent bg-clip-text">{session?.user.name}'s Roadmap</p>
```

**Review by dpaulos6 — CHANGES_REQUESTED** [review]
`session?.username` doesn't exist, it will cause the roadmap to crash or not load

**Review by dpaulos6 — CHANGES_REQUESTED** [review]
```diff
-<h1 className="text-6xl text-center font-bold mx-4"> 
-    {!session ? (
-        <p>Roadmap</p>
-        <p className="bg-gradient-to-r from-blue-600 via-green-500 to-indigo-400 inline-block text-transparent bg-clip-text">Roadmap</p>
-    ): (
-        <p>{session?.user.name}'s Roadmap</p>
-        <p className="bg-gradient-to-r from-blue-600 via-green-500 to-indigo-400 inline-block text-transparent bg-clip-text">{session?.user.name}'s Roadmap</p>
-    )}
-</h1>
+<span className="text-6xl text-center font-bold mx-4 bg-gradient-to-r from-blue-600 via-green-500 to-indigo-400 inline-block text-transparent bg-clip-text">
+  {!session ? 'Roadmap' : `${session?.user.name}'s Roadmap`}
+</span>
```

**Comment by nizzyabi** [comment]
I don't really like the gradient on the roadmap. it feels a bit much. Feel free to design the tutoring page though! it needs working.

**Pull request closed without merging**